### PR TITLE
[CI:DOCS] Bump to v2.1.0-RC1

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by mheon** [comment]
@rhatdan @vrothberg @baude @TomSweeneyRedHat PTAL

**Comment by rhatdan** [comment]
LGTM

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,471 @@
...
+  * Fix ssh-agent support
+  * APIv2:doc: Fix swagger doc to refer to volumes
+  * BATS tests: new too-many-arguments test
+  * Reformat inspect network settings
+  * Add podman network to bash command completions
+  * Fix typo in manpage for `podman auto update`.
+  * Add tests for --privileged with other flags
+  * Add JSON output field for ps
+  * V2 podman system connection
+  * wip
```
Comment (TomSweeneyRedHat) [inline changelog.txt:428]: Did we really merge a PR only with "wip" as a title?

**Review by TomSweeneyRedHat — COMMENTED** [review]
One question otherwise LGTM

File: changelog.txt (hunk 1 above)
Comment (vrothberg) [reply changelog.txt:428]: Yes :-D

**Review by vrothberg — COMMENTED** [review]
LGTM

We've not been lazy, that's for sure!

**Pull request merged**